apps/bugsy/management/commands/process_bugogram.py

```diff
@@ -37,7 +37,7 @@ class Command(BaseCommand):
     props = re.compile('^; (.*?) : (.*?)$', re.M | re.S)
     params = re.compile('%\((.*?)\)s')
 
-    def handle(self, *args, **options):
+    def handle(self, **options):
```

Comment:
What's the rationale for removing `*args`?

Reply:
I read somewhere that it's deprecated, but I forgot which version/doc that was in.

In https://github.com/django/django/blob/stable/1.11.x/django/core/management/__init__.py#L127, it shows that *args is only filled if you actually create a named argparse argument `args`. All others end up as keyword arguments now.